app/components/widgets/forms/ticket-input.js

```diff
@@ -1,8 +1,15 @@
 import Component from '@ember/component';
+import { computed } from '@ember/object';
 
 export default Component.extend({
-  classNames: ['ui', 'celled', 'stackable', 'grid', 'ticket-row'],
-
+  classNames  : ['ui', 'celled', 'stackable', 'grid', 'ticket-row'],
+  minMaxValid : computed('ticket.minPrice', 'ticket.maxPrice', function() {
+    if (this.ticket.minPrice > this.ticket.maxPrice) {
```

Comment:
Use [`gt`](http://api.emberjs.com/ember/3.10/functions/@ember%2Fobject%2Fcomputed/gt)

Reply:
Yeah, I guess that's better